Move to github.com/mholt/archives

jraddaoui opened the pull request:
The github.com/mholt/archiver repository has been archived. Upgrade temporal-activities to match archives version.

Refs https://github.com/artefactual-sdps/temporal-activities/issues/46, depends on https://github.com/artefactual-sdps/temporal-activities/pull/52.

djjuhasz reviewed (approved):
Looks good, thanks @jraddaoui. :+1:

jraddaoui committed:
Move to github.com/mholt/archives

The github.com/mholt/archiver repository has been archived.
Upgrade temporal-activities to match archives version.
